SpriteKit/MendeluSpriteKit/Screens/Game/GameViewController.swift

```diff
@@ -71,11 +71,31 @@ final class GameViewController: UIViewController {
             gameObject.keyboardDown(presses: presses)
         }
     }
+    
+    func welcomeScreen() {
+        guard
+            let scene = WelcomeScreen(fileNamed: Assets.Scenes.welcomeScreen),
+            let welcomeBackground = UIImage(named: Assets.Image.welcomeBackground)
+        else {
+            return
+        }
+        
+        scene.newGameButtonTapped = { [weak self] in
```

Comment:
`LevelScene` uses delegate pattern and here is closure 😄 I would rather use only one to keep things consistent (preferably delegate) 😄

Reply:
Done